### PR TITLE
part3: Note about data scheme

### PR DESCRIPTION
**Pull request by mand35**
A comment is added about the predefined json data scheme used in the new implementation.

### CODE REVIEW

File: docs/getting-started/part-3/custom-webservice.md [hunk 1]
```diff
@@ -200,6 +200,8 @@ This logic is implemented in the next code snipped and should replace the code c
 }
 
 ```
+Note, in the *Init* case, the pre-defined **schema** (see the `ext-departures` dataObject definition in the config) is used for the creation of an empty/dummy json file, which then gets converted to the DataFrame.
```
Comment (pgruetter) [inline docs/getting-started/part-3/custom-webservice.md]:
```suggestion
Note, in the *Init* phase, the pre-defined **schema** (see the `ext-departures` dataObject definition in the config) is used to create an empty/dummy json file, which then gets converted to the DataFrame.
```

Comment (pgruetter) [reply docs/getting-started/part-3/custom-webservice.md]: @zzeekk I think this is not strictly true. We don't create an empty Json file, we generate a DataFrame on the fly with the given schema, correct?

**Review by pgruetter — CHANGES_REQUESTED** [review]

File: docs/getting-started/part-3/custom-webservice.md (hunk 1 above)
Comment (mand35) [reply docs/getting-started/part-3/custom-webservice.md]: I am happy if we can describe it in a correct way. I only want to point out that the defined schema comes in place here. This is the important bit for me. 
Thanks for finding to correct description of the present command. 
```Scala
Seq[String]().toDF("responseString")
.select(from_json($"responseString", DataType.fromDDL(schema)).as("response"))                                                                                                                                                            
.select(explode($"response").as("record"))                                                                                                                                                                                                
.select("record.*")                                                                                                                                                                                                                       
.withColumn("created_at", current_timestamp()) 
```

File: docs/getting-started/part-3/custom-webservice.md [hunk 2]
```diff
@@ -200,6 +200,8 @@ This logic is implemented in the next code snipped and should replace the code c
 }
 
 ```
+Note, in the *Init* phase, the pre-defined **schema** (see the `ext-departures` dataObject definition in the config) is used to create an empty/dummy json file, which then gets converted to the DataFrame.
```
Comment (Geheiner) [inline docs/getting-started/part-3/custom-webservice.md]:
```suggestion
Note, in the *Init* phase, the pre-defined **schema** (see the `ext-departures` dataObject definition in the config) is used to create an empty/dummy json string, which then gets converted to the DataFrame.
```

**Review by Geheiner — CHANGES_REQUESTED** [review]

**Pull request merged**